libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -1040,7 +1040,6 @@ def add_version_header(tc):
             "name": "__cpp_lib_reference_wrapper",
             "values": {"c++26": 202403}, # P2944R3: Comparisons for reference_wrapper
             "headers": ["functional"],
-            "unimplemented": True,
```

Comment:
This should be set after the entire paper is implemented.

Reply:
I removed the setting as requested.

My understanding is that the FTM is set when a feature is complete. This PR completes comparisons for the reference wrapper, which is the first part of the paper. So it the FTM should(could) be set. 
The paper has a second (to me unrelated) part about constraining the equality of unrelated to reference wrapper types. For that part the paper defines a second FTM `__cpp_lib_constrained_equality`.
This is how I understand it. Is my thinking wrong?

Reply:
You're correct. It's rare that a paper has multiple FTMs I'm happy Barry did that. So let's indeed mark this FTM as complete.